pkgs/tools/misc/lottieconverter/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, rlottie, libpng, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "LottieConverter";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "sot-tech";
+    repo = pname;
+    rev = "r${version}";
+    hash = "sha256:11dribhqbb6rdrbr86a9sk9iwyv6097xazp31srwv3knl23v60cl";
+  };
+
+  nativeBuildInputs = [ rlottie libpng zlib ];
+
+  buildPhase = ''
+    make CONF=Release
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -v dist/Release/GNU-Linux/lottieconverter $out/bin/
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/sot-tech/LottieConverter/";
+    description = "Lottie converter utility";
+    license = licenses.lgpl21;
```

Comment:
`lgpl21` is unclear, use `lgpl21Plus` or `lgpl21Only`. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.

Reply:
Will switch to `lgpl21Plus`, as the license contains ["or (at your option) any later version."](https://github.com/sot-tech/LottieConverter/blob/master/lottie_export.cpp#L7)